<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>271</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <action name="action1">
   <property name="text">
    <string>Delete selected row</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>Insert row</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>Insert column</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
